web/json_value_test.go

```diff
@@ -88,6 +88,46 @@ func TestConvertJSONAttributeValue(t *testing.T) {
 			valueJSON: `[true, 0, "true"]`,
 			wantValue: []bool{true, false, true},
 		},
+		"unix": {
+			attribute: &framework.AttributeConfig{
+				ExternalId: "a",
+				Type:       framework.AttributeTypeDateTime,
+			},
+			valueJSON: `"1706041056"`,
+
+			opts:      &jsonOptions{dateTimeFormats: []DateTimeFormatWithTimeZone{{"sgnl-unix-ts", false}, {"2006-01-02", false}}},
```

Comment:
This is not required if we're testing the parsing of UNIX time.
```suggestion
			opts:      &jsonOptions{dateTimeFormats: []DateTimeFormatWithTimeZone{{"sgnl-unix-ts", false}}},
```